shrinkpdf.sh

```diff
@@ -36,7 +36,7 @@ shrink() {
   gs \
     -q -dNOPAUSE -dBATCH -dSAFER \
     -sDEVICE=pdfwrite \
-    -dCompatibilityLevel=1.3 \
```

Comment:
v1.3 was probably setup to encompass modern (2005+) PDFs. Providing compatibility level equals to that of the input file is a better idea. Kudos!